src/components/common/PeopleAndRoles/PeopleAndRoles.vue

```diff
@@ -198,9 +201,26 @@ export default class PeopleAndRoles extends Mixins(CommonMixin, DateMixin, OrgPe
     return this.getEntitySnapshot?.orgPersons || []
   }
 
-  /** True if we have at least 1 Director. */
-  get dirValid (): boolean {
-    return this.hasRole(RoleTypes.DIRECTOR, 1, CompareModes.AT_LEAST)
+  /** True when the required proprietor count is met. */
+  get haveRequiredProprietor (): boolean {
+    return this.hasRole(RoleTypes.PROPRIETOR, 1, CompareModes.EXACT)
+  }
```

Comment:
Moved here from store getters, and refactored to use local hasRole().

Ditto getter below.